fibers/posix-clocks.scm.in

```diff
@@ -37,9 +37,11 @@
 
             clock-getres
             clock-gettime
-
+            is-osx?
             clock-nanosleep))
 
+(define is-osx? (string=? "Darwin" "@platform@"))
```

Comment:
I think it would be better to avoid having `is-osx` even defined and use a specific file for that (e.g. `posix-darwin-clocks.scm`, `posix-linux-clocks.scm`... etc.). Similar to what's done here: #27 . I think it would allow cleaner cross-platform implementations. Then, you could have the code in fibers.scm in this darwin specific file:

```
(if is-osx?
  (eval-when (eval load compile)
  (dynamic-call "init_affinity"
                (dynamic-link (extension-library "affinity"))))
)
```

Reply:
Resolving see... https://github.com/aconchillo/fibers/tree/libevent